bundles/org.openhab.binding.plugwise/src/main/java/org/openhab/binding/plugwise/internal/PlugwiseUtils.java

```diff
@@ -102,7 +100,18 @@ public static void stopBackgroundThread(@Nullable Thread thread) {
     }
 
     public static String upperUnderscoreToLowerCamel(String text) {
-        String upperCamel = StringUtils.remove(WordUtils.capitalizeFully(text, new char[] { '_' }), "_");
+        final String delimiter = "_";
+        String upperCamel = "";
+        for (String str : text.split(delimiter)) {
+            String properlyCapitalized = "";
+            if (str.length() > 0) {
+                properlyCapitalized = str.substring(0, 1).toUpperCase();
+            }
+            if (str.length() > 1) {
+                properlyCapitalized = str.substring(1).toLowerCase();
```

Comment:
I think this introduces a bug:

`HELLO_WORLD` will result in `elloorld` instead of `helloWorld`

```suggestion
                properlyCapitalized += str.substring(1).toLowerCase();
```

Reply:
I don't know if the compiler will optimize it to use a `StringBuilder` automatically... if not, explicitly using it would also be better because the concatenation is done in a for loop.

Reply:
Fxid + stringbuilder + test